server/sonar-web/src/main/js/apps/quality-profiles/utils.ts

```diff
@@ -65,8 +65,14 @@ export function createFakeProfile(overrides?: any) {
   };
 }
 
-export function isStagnant(profile: Profile) {
-  return moment().diff(moment(profile.userUpdatedAt), 'years') >= 1;
+export function isStagnant(profile: Profile): boolean {
+  if (profile.userUpdatedAt) {
+    const updateDate = new Date(profile.userUpdatedAt);
+    if (isValidDate(updateDate)) {
+      return new Date().getFullYear() - updateDate.getFullYear() >= 1;
```

Comment:
this is wrong: I have a QP which was updated on December 12, 2016, and it's marked as stagnant